rust/perspective-viewer/src/ts/migrate/2-6-1.ts

```diff
@@ -19,12 +19,12 @@ import { cmp_semver, parse_semver } from "../migrate";
  * @returns
  */
 export default function migrate_2_6_1(old, options) {
-    if (cmp_semver(old.version, "2.7.0")) {
-        return;
+    if (cmp_semver(old.version, "2.6.1")) {
+        return old;
     } else if (options.warn) {
         console.warn("Migrating from 2.6.1");
     }
-    old.version = parse_semver("2.7.0");
+    old.version = parse_semver("2.6.1");
```

Comment:
This line is inert (because it is overwritten by `assure_latest()`), but if it weren't it would be wrong. If `old == 2.6.1`, the migration is applied (conditional on line 22 is false) but then the version field is set back to `2.6.1` again (line 27). 

This specific script functionally upgrades _from_ `2.6.1` _to_ `2.7.0` (and presumably beyond), ergo `old.version` should be some version after `2.6.1` (which we wouldn't know at the time of authoring the migration).